backend/apollo/elections/views.py

```diff
@@ -99,6 +101,34 @@ def get_permissions(self) -> List[BasePermission]:
         return [perm() for perm in permission_classes]
 
 
-class VoteViewSet(mixins.CreateModelMixin, viewsets.GenericViewSet):
+class VoteViewSet(viewsets.GenericViewSet):
     queryset = Vote.objects.all()
     serializer_class = VoteSerializer
+
+    def create(self, request):
+        try:
+            answer_id_ciphertext = request.data["answer"]
+            election_id = request.data["election"]
+        except KeyError as e:
+            return Response(f"Key {e} not found", status=status.HTTP_400_BAD_REQUEST)
```

Comment:
I believe you could override `create()` inside `VoteSerializer`. Then you could validate the request using `validate()` as usual